prometheus/build.gradle

```diff
@@ -42,6 +42,7 @@ test {
 configurations.all {
     resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib:1.6.0"
     resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib-common:1.6.0"
+    resolutionStrategy.force "org.junit.jupiter:junit-jupiter:5.6.2"
```

Comment:
Is it safe to use alpha releases? Curious as to whether it is better to force it here or to upgrade mockwebserver to a version that doesn't bring in the older version

https://mvnrepository.com/artifact/com.squareup.okhttp3/mockwebserver

Reply:
It is better to upgrade but 4.10 has the same junit CVE and 5+ are alpha candidates.